libraries/lib-utility/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ set( SOURCES
    GlobalVariable.h
    IteratorX.cpp
    IteratorX.h
+   MathApprox.h
```

Comment:
I am confused. Why `lib-utility` instead of `lib-math`?

Reply:
`lib-math` depends on `lib-soxr`. I'm reluctant to let the time-and-pitch depend on this mainly because it compiles PFFFT functions which are also found in `lib-fft`, another dependency of time-and-pitch.

Reply:
And why is it a problem? It doesn't introduce a circular dependency.

Reply:
Not a circular dependency, but how can you be confident that it's the `lib-fft` symbols that will be used?